pages/nurses/[id]/index.js

```diff
@@ -37,19 +41,19 @@ export default function NursePage() {
       `Are you sure you want to fire ${data?.name}?`
     );
     if (shouldDelete) {
-      const response = await fetch(`/api/nurses/${id}`, { method: "DELETE" });
+      const response = await fetch(`/api/nurses/${id}/`, { method: "DELETE" });
```

Comment:
Do you need the trailing slash? Just wondering. If you experience errors sometimes, you might need to drop it.